examples/e31_indefinite_transitions.clj

```diff
@@ -0,0 +1,248 @@
+                   pause-1s {:fx/type :pause-transition
+                             :duration [1 :s]}
+                   rotate-right {:fx/type :rotate-transition
+                                 :duration [0.5 :s]
+                                 :from-angle 0
+                                 :to-angle 185
+                                 :interpolator :ease-both}
+                   translate-right {:fx/type :translate-transition
+                                    :duration [0.5 :s]
+                                    :from-x 0
+                                    :to-x 75
+                                    :interpolator :ease-both}
+                   rotate-left (switch-props rotate-right :from-angle :to-angle)
+                   translate-left (switch-props translate-right :from-x :to-x)
+                   parallel-dance-right {:fx/type :parallel-transition
+                                         :children [rotate-right
+                                                    translate-right]}
+                   parallel-dance-left {:fx/type :parallel-transition
+                                        :children [rotate-left
+                                                   translate-left]}]
+               {:fx/type :parallel-transition
+                :cycle-count :indefinite
+                :children [{:fx/type :fade-transition
+                            :from-value 1.0
+                            :to-value 0.3
+                            :auto-reverse true
+                            :duration [0.5 :s]
+                            :cycle-count 6}
+                           {:fx/type :sequential-transition
+                            :children [pause-1s
+                                       parallel-dance-right
+                                       pause-1s
+                                       parallel-dance-left]}]})]])
+
+(defn view [{_ :state}]
+  {:fx/type :stage
+   :showing true
+   :always-on-top true
+   :width 600
+   :height 500
+   :scene {:fx/type :scene
+           :root {:fx/type on-grid
+                  :rows 5
+                  :columns 2
+                  :children (mapv
+                              (fn [[header transition]]
+                                (with-header
+                                  header
+                                  {:fx/type start-transition-on
+                                   :transition transition
+                                   ; run each animation on a small rectangle
+                                   :desc {:fx/type :rectangle
+                                          :width 20
+                                          :height 50}}))
+                              indefinite-animations)}}})
+
+(def renderer
+  (fx/create-renderer
+    :middleware (fx/wrap-map-desc (fn [state]
+                                    {:fx/type view
+                                     :state state}))))
+
+(fx/mount-renderer *state renderer)
```

Comment:
I noticed you don't use state at all, in that case there is no need to introduce state/renderer in example, just use `fx/create-component`:
```clj
(fx/on-fx-thread
  (fx/create-component
    {:fx/type :stage
     :showing true
     :always-on-top true
     :width 600
     :height 500
     :scene {:fx/type :scene
             :root {:fx/type on-grid
                    :rows 5
                    :columns 2
                    :children (mapv
                                (fn [[header transition]]
                                  (with-header
                                    header
                                    {:fx/type start-transition-on
                                     :transition transition
                                     ; run each animation on a small rectangle
                                     :desc {:fx/type :rectangle
                                            :width 20
                                            :height 50}}))
                                indefinite-animations)}}}))
```

Reply:
Moved the state stuff to a comment; I think it's always nice to be able to use state to play with the examples.